django/contrib/gis/geos/collections.py

```diff
@@ -134,8 +136,11 @@ class MultiPolygon(GeometryCollection):
 
     @property
     def cascaded_union(self):
-        "Returns a cascaded union of this MultiPolygon."
-        return GEOSGeometry(capi.geos_cascaded_union(self.ptr), self.srid)
```

Comment:
Just to be sure, the new implementation is guaranteed to behave the same as the old one? Maybe the docs could call this out "which behaves exactly the same."